libact/query_strategies/multiclass/tests/test_hierarchical_sampling.py

```diff
@@ -0,0 +1,52 @@
+""" UncertaintySampling test
+"""
+import unittest
+
+import numpy as np
+from sklearn import datasets
+from sklearn.utils import shuffle
+
+from libact.base.dataset import Dataset
+from libact.models import SVM
+from libact.query_strategies import UncertaintySampling
+from libact.query_strategies.multiclass import HierarchicalSampling as HS
+
+
+def run_qs(ds, qs, y_truth, quota):
+    ret = []
+    for _ in range(quota):
+        ask_id = qs.make_query()
+        ds.update(ask_id, y_truth[ask_id])
+        y_pred = qs.report_all_label()
+        score = sum(y_pred == y_truth) / len(y_truth)
+        ret.append(score)
+    return ret
+
+
+class HierarchicalSamplingTestCase(unittest.TestCase):
```

Comment:
Maybe add a test case to make sure the sequence of ask_id make_query returns remains fix?